sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_general_document_async.py

```diff
@@ -7,18 +7,14 @@
 # --------------------------------------------------------------------------
 
 """
-FILE: sample_analyze_document_async.py
+FILE: sample_analyze_general_document_async.py
```

Comment:
Just going to move the naming conversation over here. So you suggest prebuilt instead of general? I'll update it.

Reply:
Unless we can find a better name, I think "prebuilt" tells you more about what it is than "general" does. Not sure what other languages are doing though